tests/unit/lib/intrinsic_resolver/test_intrinsic_resolver.py

```diff
@@ -0,0 +1,1347 @@
+import json
+from copy import deepcopy
+from pathlib import Path
```

Comment:
pathlib is not in the Python2.7 standard library. You will need to do 

```
try:
    import pathlib
except ImportError:
    import pathlib2 as pathlib
```

to make the Py2.7 tests pass.